### DISCUSSION TITLE
Run button doesnt work and I get error messages

### DISCUSSION THREAD

**Discussion by Argusf28** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#38] [Q&A] [answered]
Hi, I cant seem to make the scrip work.
The run button does not work like such, its just a regular element I can doble click and edit.
The time driven trigger does not auto excecute and manually I get this error message:

Exception: Unexpected error while getting the method or property getFolderById on object DriveApp.
    at uploadToDrive(bundle:273:27)
    at [unknown function](bundle:438:16)
    at fetch(bundle:429:10)
    at [unknown function](bundle:675:17)
    at batchFetch(bundle:673:8)

What am I doing wrong? Thanks in advance!

**Comment by chriskyfung** [comment, marked as answer]
Thanks for your reporting!

The run button did not work because I forgot to assign a script to it.
This issue has been fixed in the template v3.

Regarding to the error message, I reproduce it if an incorrect Folder ID is filled in the Settings page.
You may leave the cell blank instead to test if the time driven trigger work properly.

**Reply by Argusf28** [reply]
Thanks so much! Its working now (but Im having the same issue with the "create badges" button)
However, the files keep getting downloaded to the root folder, even with a folder ID on Destination Folder ID.
Still, this was a God send for me.

**Reply by chriskyfung** [reply]
You can make the "create badges" button work by assigning it with the script `createBadges`.

To save the downloaded files to a destination folder, you should put a folder starts with a 1 followed by 32 [base-64](https://en.wikipedia.org/wiki/Base64) encoded digits,  the one that appears in the URL bar when you open a folder in your Google drive, the thing after `https://drive.google.com/drive/u/0/folders/<this part>`.

**Reply by Argusf28** [reply]
Thanks for your reply!

I was using that exact string of numbers on the destination folder ID, yet it still downloads everything to my root drive, weird.

As for the Badges, since im just an average user with no knowledge on these things, couldnt find on google a way to make this work. Pretty sure is super simple, but Im a complete newbie here so, I guess I will just check from time to time if everything is working haha.

**Reply by chriskyfung** [reply]
Make a copy of the updated Google Sheet template to your Google Drive.

<div style="padding-left: 1.5em">
  <a href="https://docs.google.com/spreadsheets/d/1VucEhAUn-mq2Z38QrVGa9GdFrmHY4UCPMCi6mrWRnQI/copy">
    <img src="https://img.shields.io/badge/Google%20Sheet-34a853?logo=googlesheets&logoColor=white" height="28" alt="">
  </a>
</div>
